### ISSUE TITLE
Is the png of mesh necessary?

### ISSUE THREAD

**Issue by BIT-TYJ** [megapose6d/megapose6d#58]
Hello, I try a novel object with the mesh (.obj) below, whose center is at the origin, measured in millimeters.
![85c5e87543ed9c069067385ed6fa58c](https://github.com/megapose6d/megapose6d/assets/79152152/a2fccc22-4db5-4ba6-9a33-ea9280b4807c).
The result is as follows:
![f8c5208f9c9efa2557d5e73733f58fb](https://github.com/megapose6d/megapose6d/assets/79152152/15a25f1c-a96d-43b5-b76d-da05ce0a5a2f)

The mesh doesn't have the corresponding .png, I want to know where is the problem?

**Comment by BIT-TYJ** [comment]
Besides, the mesh has no bottom, does it influence the effect?

**Comment by Brohoya** [comment]
Hi @BIT-TYJ, did you manage to get a rendering from your mesh without a texture ? I face the same kind of output (even if my mesh is fully closed). I tried to put a random png but I get the same result (which is just my image with a white filter)...
![mesh_overlay](https://github.com/megapose6d/megapose6d/assets/84842676/67bad864-294f-43e7-aafd-f85d929aba35)

**Comment by dqj5182** [comment]
I have run demo with their data but without png of mesh and here is its result!
Not sure how to interpret this result but seem different than results posted above.

![all_results](https://github.com/megapose6d/megapose6d/assets/56655511/5134a829-cb94-412e-9f12-b9bd50e0d327)

**Comment by mmkolbe** [comment]
Yes, the texture map is need. I just ran a lot of tests here and in fact it is of huge importance. I tried two things: a) remove the texture map and b) messing the texture map.
The PLY file has the following lines at the begining:
```
ply
format ascii 1.0
comment TextureFile hope_000002.png
element vertex 8725
property float x
property float y
...
```
For test a) I remove the line, and the inference can't locate the object;
For the test b), i changed from this:

![Image](https://github.com/user-attachments/assets/cc47e1fe-6696-4922-a343-7d21f1ee8e4d)

To this:

![Image](https://github.com/user-attachments/assets/8944c87a-5471-4f41-87dc-5173e6e9148a)
And this is the result:

![Image](https://github.com/user-attachments/assets/11f8a55b-fc48-4f82-876f-0d84564761c6)
Instead of the original result:

![Image](https://github.com/user-attachments/assets/cc618124-c448-41e4-99b9-2fa7274be193)

**Comment by lalaLQW** [comment]
> 您好，我尝试创建一个名为“新物体”的网格模型（.obj），其中心位于原点，尺寸以毫米为单位。 ![85c5e87543ed9c069067385ed6fa58c](https://github.com/megapose6d/megapose6d/assets/79152152/a2fccc22-4db5-4ba6-9a33-ea9280b4807c)结果 如下： ![f8c5208f9c9efa2557d5e73733f58fb](https://github.com/megapose6d/megapose6d/assets/79152152/15a25f1c-a96d-43b5-b76d-da05ce0a5a2f)
> 
> 网格模型没有对应的.png文件，我想知道问题出在哪里？

你可以在blender里面烘焙出一个纹理文件，并将其导入到ply文件里面。纹理文件是非常重要的。我在测试我的新物体时（对称扁平结构），如果没有纹理结构，输出的可视化结果直接不对，但是加上纹理文件之后，位姿估计的状态会很精准和稳定
